build.xml

```diff
@@ -147,35 +165,16 @@
 
     <!-- Target: init -->
     <target name="init" description="Initialize the repo for development">
-        <!-- Link search_api_federated_solr:master (D8) into the src/ directory so the code is easy to find. -->
-        <mkdir dir="src" />
-        <delete file="src/search_api_federated_solr" />
-        <symlink target="../web/d8/docroot/modules/contrib/search_api_federated_solr" link="src/search_api_federated_solr" />
-
-        <!-- Link search_api_field_map:master (D8) into the src/ directory so the code is easy to find. -->
         <mkdir dir="src" />
-        <delete file="src/search_api_field_map" />
-        <symlink target="../web/d8/docroot/modules/contrib/search_api_field_map" link="src/search_api_field_map" />
+        <delete dir="src/search_api_federated_solr" />
+        <exec command="git clone --branch=8.x-2.x git@github.com:palantirnet/search_api_federated_solr.git" dir="src" checkreturn="true" />
```

Comment:
I have the same concern here as with above: this command sometimes gets re-run, and then developers could unintentionally and irreversibly trash any local development on the modules -- even if they've committed it to git but haven't pushed it yet.